frontend/include/chpl/types/BoolType.h

```diff
@@ -55,41 +55,26 @@ class BoolType final : public PrimitiveType {
  public:
   ~BoolType() = default;
 
-  static const BoolType* get(Context* context, int bitwidth);
+  static const BoolType* get(Context* context);
 
   int bitwidth() const override {
-    if (bitwidth_ == 0) return 8;
-    return bitwidth_;
+    return 8; // Chapel doesn't guarantee this bidwth, so this seems fragile...
   }
 
   bool isDefaultWidth() const override {
-    return bitwidth_ == defaultBitwidth();
+    return true;
   }
 
   const char* c_str() const override {
-    switch (bitwidth_) {
-      case 0:
-        return "bool";
-      case 8:
-        return "bool(8)";
-      case 16:
-        return "bool(16)";
-      case 32:
-        return "bool(32)";
-      case 64:
-        return "bool(64)";
-      default:
-        CHPL_ASSERT(false && "bool bit width case not handled");
-        return "bool(<unknown>)";
-    }
+    return "bool";
   }
 
   /**
    Returns `true` if this is the type `bool` which is distinct from
    `bool(8)`.
    */
   bool isDefaultBool() const {
-    return bitwidth_ == 0;
+    return true;
```

Comment:
This query isn't necessary anymore, is it?

Reply:
I hadn't noticed, but it looks like you're correct.  Removing it.